rocAL/rocAL/include/api/rocal_api_data_transfer.h

```diff
@@ -48,5 +48,8 @@ extern "C"  RocalStatus   ROCAL_API_CALL rocalCopyToOutputTensor(RocalContext ro
 /// \param output_images The buffer that will be filled with output images with set_output = True
 extern "C" void ROCAL_API_CALL rocalSetOutputs(RocalContext p_context, unsigned int num_of_outputs, std::vector<RocalImage> &output_images);
 
+extern "C"  RocalStatus   ROCAL_API_CALL rocalExternalSourceFeedInput(RocalContext rali_context,std::vector<std::string> input_images, std::vector<int> labels,
```

Comment:
Can u remove the extra spaces after RocalStatus? I see that its the format used in the other functions but if they see this, they may ask us to change the spacing for those functions as well